llvm/lib/Target/SPIRV/SPIRVEmitIntrinsics.cpp

```diff
@@ -182,7 +182,7 @@ class SPIRVEmitIntrinsics
 
   bool deduceOperandElementTypeCalledFunction(
       CallInst *CI, SmallVector<std::pair<Value *, unsigned>> &Ops,
-      Type *&KnownElemTy);
+      Type *&KnownElemTy, bool &Uncomplete);
```

Comment:
nitpick (probably?): the name implies the procedure that `deduceOperandElementTypeCalledFunction` does can be canceled or postponed. Is it the case?